src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartDataGroupingsConfiguration.test.tsx

```diff
@@ -321,4 +321,57 @@ describe('ChartDataGroupingsConfiguration', () => {
     expect(await screen.findByText('Cannot save chart')).toBeInTheDocument();
     expect(screen.getByText('Options tab is invalid')).toBeInTheDocument();
   });
+
+  test('making a data grouping change and submitting the form', async () => {
+    const handleChange = jest.fn();
+    const handleSubmit = jest.fn();
+
+    const { user } = render(
+      <ChartDataGroupingsConfiguration
+        meta={testTable.subjectMeta}
+        map={{
+          dataSetConfigs: [
+            {
+              dataGrouping: defaultDataGrouping,
+              dataSet: {
+                filters: ['ethnicity-major-chinese', 'state-funded-primary'],
+                indicator: 'authorised-absence-sessions',
+                timePeriod: '2014_AY',
+              },
+            },
+            {
+              dataGrouping: defaultDataGrouping,
+              dataSet: {
+                filters: ['ethnicity-major-chinese', 'state-funded-primary'],
+                indicator: 'authorised-absence-sessions',
+                timePeriod: '2015_AY',
+              },
+            },
+          ],
+        }}
+        onSubmit={handleSubmit}
+        onChange={handleChange}
+      />,
+    );
+
+    await user.click(screen.getAllByRole('button', { name: 'Edit' })[0]);
+
+    expect(await screen.findByRole('dialog')).toBeInTheDocument();
+    expect(screen.getByText('Edit groupings')).toBeInTheDocument();
+
+    await user.click(screen.getByLabelText('Quantiles'));
+    await user.click(screen.getByRole('button', { name: 'Done' }));
+
+    await waitFor(() => {
+      expect(handleChange).toHaveBeenCalledTimes(1);
+    });
+
+    await user.click(
+      screen.getByRole('button', { name: 'Save chart options' }),
+    );
+
+    await waitFor(() => {
+      expect(handleSubmit).toHaveBeenCalledTimes(1);
```

Comment:
We should check what `handleSubmit` has been called with

Reply:
Added test `Data groupings tab edits submit with correct values` to **ChartBuilder.test.tsx**